### PR TITLE
Code review shenanigans

### PR DESCRIPTION
**Pull request by pkoppol**
Hi Michael, 

**Purpose:** Some small changes to be code reviewed, and rejected! :) 

**Feedback Type:** Please leave feedback in the comments themselves.

### CODE REVIEW

File: algo_strategy.py [hunk 1]
```diff
@@ -107,7 +106,7 @@ def starter_strategy(self, game_state):
 
 
     def use_extra_cores(self, game_state):
-        destructor_spawn_location = [[9,10], [11,10],[13,10], [15,10], [17,10], [19,10]]
+        destructor_spawn_location = [[8,9], [10,9],[12,9], [14,9], [16,9], [18,9]]
```
Comment (pkoppol) [inline algo_strategy.py]: There shouldn't be magic numbers here.

**Review by pkoppol — COMMENTED** [review]
I agree

File: algo_strategy.py (hunk 1 above)
Comment (michaelhuang14) [inline algo_strategy.py:109]: No magic numbers allowed >:(

**Review by michaelhuang14 — APPROVED** [review]
Important change, necessary for overall coherence

**Pull request merged**